gui/window.go

```diff
@@ -106,6 +109,33 @@ func (window *Window) CreateHeaderBar() error {
 	return nil
 }
 
+// SetRelativeSize queries monitor size and sets a relative size for the given window
+func SetRelativeSize(win *Window, w float32, h float32) error {
+
+	screen, err := gdk.ScreenGetDefault()
+	if err != nil {
+		return err
+	}
+
+	dpy, err := screen.GetDisplay()
+	if err != nil {
+		return err
+	}
+
+	mon, err := dpy.GetPrimaryMonitor()
+	if err != nil {
+		return err
+	}
+
+	rect := mon.GetGeometry()
+
+	win.handle.SetDefaultSize(
```

Comment:
I like that this will scale for smaller resolutions, but I think our current size is about the largest we should make the install screen. Can we calculate the relative size first, and then set the ceiling at our current defaults?
    


```suggestion
        relW := int(w*float32(rect.GetWidth())
        relH := int(h*float32(rect.GetHeight())
        if relW > WindowWidth || relH > WindowHeight {
             window.handle.SetDefaultSize(WindowWidth, WindowHeight)
        } else {
             window.handle.SetDefaultSize(relW, relH)
        }     
```

Reply:
Sure, that sounds fine approach to me. Will fix and re-test.

Reply:
OK, what I ended up doing looks a bit different but should do the same thing. I'm comparing to the actual monitor size and if we don't fit then I use the relative size.